sbws/lib/resultdump.py

```diff
@@ -215,15 +216,18 @@ def __init__(self, fingerprint, nickname, address, master_key_ed25519,
             self.consensus_bandwidth = consensus_bandwidth
             self.consensus_bandwidth_is_unmeasured = \
                 consensus_bandwidth_is_unmeasured
+            self.consensus_count = consensus_count
```

Comment:
I think this new field should be documented somewhere.  You should probably add documentation, unless some later commit already does that.